src/pretix/base/models/orders.py

```diff
@@ -3098,7 +3098,10 @@ class CartPosition(AbstractPosition):
         verbose_name=_("Expiration date"),
         db_index=True
     )
-
+    max_extend = models.DateTimeField(
```

Comment:
Only 80% happy with the naming, I would have chosen a more direct parallel to "expires", but "max_expires" can also be confusing… Not something I have strong feelings about

Reply:
I also considered `max_expires` but I thought it's too confusing. Do you think `max_expires` is better? Or maybe `extend_expiry_limit`? I'm not really attached to `max_extend` either, and I think it's unique enough the I can easily change the name with a replace-all :D